### ISSUE TITLE
Medicaiton Dispense issue

### ISSUE THREAD

**Issue by jddamore** [jddamore/ccda-search#24]
https://hl7.org/ccdasearch/templates/2.16.840.1.113883.10.20.22.4.18.html

@brettmarquard says this looks different than PDF

**Comment by brettmarquard** [comment]
Close -- I think the valueSet is in wrong spot

**Comment by jddamore** [comment]
Agree. Second pass of fix here: https://github.com/jddamore/ccda-search/commit/39d107d29874ecf03225cdf97b2b0812571cdeaa

**Comment by brettmarquard** [comment]
Looks good.

**Comment by jddamore** [comment]
Thanks Brett!
